val/src/avs_pe.c

```diff
@@ -66,22 +66,17 @@ val_pe_execute_tests(uint32_t level, uint32_t num_pe)
   status |= c013_entry(num_pe);
   status |= c014_entry(num_pe);
   status |= c015_entry(num_pe);
-
-  if (level > 2) {
-      status |= c016_entry(num_pe);
-      status |= c017_entry(num_pe);
-  }
+  status |= c016_entry(num_pe);
+  status |= c017_entry(num_pe);
 
   if (level > 5) {
       status |= c018_entry(num_pe);
   }
 
-  if (level > 3) {
```

Comment:
These tests are applicable for levels 4 and above. Not required to be executed for level 3. So, this change is not required.